eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -49,40 +49,90 @@ param(
   [string]$PRBody = $PRTitle,
 
   [Parameter(Mandatory = $false)]
-  [string]$PRLabels
+  [string]$PRLabels,
+
+  [Parameter(Mandatory = $false)]
+  [string]$UserReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$TeamReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$Assignees,
+
+  $PRNumber
 )
 
+$baseURI = "https://api.github.com/repos"
+function SplitMembers ($membersString)
+{
+  return @($membersString.Split(",") | % { $_.Trim() } | ? { return $_ })
+}
+
+$userAdditions = SplitMembers -membersString $UserReviewers
+$teamAdditions = SplitMembers -membersString $TeamReviewers
+$labelAdditions = SplitMembers -membersString $PRLabels
+$assigneeAdditions = SplitMembers -membersString $Assignees
+
 $headers = @{
   Authorization = "bearer $AuthToken"
 }
 
-$query = "state=open&head=${PROwner}:${PRBranch}&base=${BaseBranch}"
-
-function AddLabels([int] $prNumber, [string] $prLabelString)
-{
-  # Adding labels to the pr.
-  if (-not $prLabelString) {
-    Write-Verbose "There are no labels added to the PR."
-    return
+function AddMembers($apiURI, $memberName, $additionSet, $body) {
```

Comment:
The name AddMembers doesn't at all signal to me that this is going to make a rest call. Lets try to update the name to help make that clearer.